web_widget_datepicker_options/__openerp__.py

```diff
@@ -0,0 +1,36 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright (C) 2015 Savoir-faire Linux
+#    (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    "name": 'web_widget_datepicker_options',
+    "version": "1.0",
+    "depends": [
+        'base',
+        'web',
+    ],
+    "js": [
+        'static/src/js/datepicker.js',
```

Comment:
datepicker.js will never be loaded with this type of reference in 8.0.

In 8.0 the javascript files must be referenced in a qweb template like in: https://github.com/OCA/web/blob/8.0/web_dialog_size/view/qweb.xml#L6